src/odemis/driver/phenom.py

```diff
@@ -202,63 +203,63 @@ def __init__(self, name, role, children, host, username, password, phenom_gui=Fa
         except ValueError:
             logging.warning("Phenom version could not be retrieved")
 
-        # Lock in order to synchronize all the child component functions
```

Comment:
Every child in this class is really a delegated child, as you noted by not modifying the code. => Revert the changes to the comments and messages.